types/relay-compiler/index.d.ts

```diff
@@ -7,66 +7,6 @@
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 3.4
 
-import * as ASTConvert from './lib/core/ASTConvert';
-import { CompilerContext } from './lib/core/CompilerContext';
-import * as Parser from './lib/core/RelayParser';
-import * as Printer from './lib/core/IRPrinter';
-import * as IRTransforms from './lib/core/RelayIRTransforms';
-import * as IRVisitor from './lib/core/IRVisitor';
-import * as SchemaUtils from './lib/core/GraphQLSchemaUtils';
+declare const binary: string;
```

Comment:
```suggestion
declare const binary: string | null;
```
https://github.com/facebook/relay/blob/main/packages/relay-compiler/index.js#L30
so the test should expect symbol to be"
```ts
// $ExpectType string | null
```